pipeline/inputs/kafka.md

```diff
@@ -0,0 +1,75 @@
+# Kafka
+
+Kafka input plugin allows to subscribe to one or more topics and collect messages from an [Apache Kafka](https://kafka.apache.org/) service. This plugin use the official [librdkafka C library](https://github.com/edenhill/librdkafka) \(built-in dependency\)
+
+## Configuration Parameters
+
+| Key | Description | default |
+| :--- | :--- | :--- |
+| brokers | Single of multiple list of Kafka Brokers, e.g: 192.168.1.3:9092, 192.168.1.4:9092. |  |
+| topics | Single entry or list of topics separated by comma \(,\) that Fluent Bit will subscribe to. |  |
+| client\_id | Client id passed to librdkafka. | |
+| group\_id | Group id passed to librdkafka. | fluent-bit |
+| poll\_ms | Kafka brokers polling interval in milliseconds. | 500 |
+| rdkafka.{property} | `{property}` can be any [librdkafka properties](https://github.com/edenhill/librdkafka/blob/master/CONFIGURATION.md) |  |
+
+## Getting Started
+
+In order to subscribe/collect messages from Apache Kafka, you can run the plugin from the command line or through the configuration file:
+
+### Command Line
+
+The **kafka** plugin, can read the parameters from the command line in two ways, through the **-p** argument \(property\), e.g:
```

Comment:
`in two ways`? There's only one way...

Reply:
Copied from out_kafka, fixing in both...